scalableminds/webknossos
# [WIP] run diff_schema.js on startup (fix)

This makes the `diff_schema.js` run on wk startup, using [trireme](https://github.com/apigee/trireme).

Some compatibility changes were necessary, as trireme is built on old JS and node versions.

### Steps to test:
- Check in CI (e.g. smoke-test), that `diff_schema.js` is successfully run on startup.

### Issues:
- fixes #3161

------
